clang-tools-extra/test/clang-doc/test-path-abs.cpp

```diff
@@ -0,0 +1,7 @@
+// RUN: rm -rf %t
+// RUN: mkdir %t
+// RUN: echo "CHECK: var RootPath = \"%t\";" > %t/check.txt
+// RUN: cp "%s" "%t/test.cpp"
```

Comment:
what is this doing? `CHECK` lines belong in the test file, generally. If you need to split them up somehow, there is the `split-file` utility (https://llvm.org/docs/TestingGuide.html#extra-files). But in this case, you don't need that at all, since these lines can be in this file